arangod/Cluster/Maintenance.cpp

```diff
@@ -524,12 +523,19 @@ arangodb::Result arangodb::maintenance::executePlan (
   report.add(VPackValue(ACTIONS));
   { VPackArrayBuilder a(&report);
     // enact all
-    for (auto const& action : actions) {
+    for (auto& action : actions) {
       LOG_TOPIC(DEBUG, Logger::MAINTENANCE)
-        << "adding action " << action << " to feature ";
+        << "phase one adding action " << action << " to feature ";
```

Comment:
Should LOG_TOPIC occur after potential add of shardVersion?  Otherwise how will logs help if there is an error with that parameter?

Reply:
Good point